core/deployment/src/main/java/io/quarkus/deployment/steps/NativeImageAutoFeatureStep.java

```diff
@@ -63,6 +66,7 @@
             "org.graalvm.nativeimage.impl.RuntimeClassInitializationSupport",
             "rerunInitialization", void.class, Class.class, String.class);
     static final String RUNTIME_REFLECTION = RuntimeReflection.class.getName();
+    static final String JNI_RUNTIME_ACCESS = JNIRuntimeAccess.class.getName();
```

Comment:
Maybe I should just give the class name as a string, avoiding the dependency? I'll try that.

Reply:
So that's what I ended up doing. Makes the Kafka Streams quickstart pass for me.